packages/compiler/src/collector/collect.ts

```diff
@@ -1,62 +1,54 @@
 import {
-  Project,
   Node,
-  SyntaxKind,
   JsxOpeningElement,
   JsxSelfClosingElement,
   JsxAttribute,
 } from "ts-morph";
 import { match } from "ts-pattern";
-import { decode } from "./decode";
 import { handleJsxExpression } from "./expression";
-import { extractPseudoAttribute } from "./pseudo";
+import * as types from "../types";
 
 export const collectPropsFromJsx = (
   node: JsxOpeningElement | JsxSelfClosingElement
-) => {
+): Record<string, types.Value> => {
   const jsxAttributes = node.getAttributes();
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any -- FIXME
-  const extracted: Record<string, any> = {};
+  const extracted: Record<string, types.Value> = {};
+
   jsxAttributes.forEach((jsxAttribute) => {
     if (Node.isJsxAttribute(jsxAttribute)) {
-      const propName = jsxAttribute.getNameNode().getFullText();
-      let propValue;
-      // If the propName starts with underscore, use extractPseudoAttribute
-      if (propName.trim().startsWith("_")) {
-        propValue = extractPseudoAttribute(jsxAttribute);
```

Comment:
Note: now `extractPseudoAttribute` is gone and we just use the same function, `extractAttribute` for both

Reply:
With the current changes, the logic for determining the pseudo prop has been removed. This is unsafe, so it is necessary to add similar determination logic in `handleJsxExpression` by tracing back to the parent node from the expression.